prepare_data.py

```diff
@@ -0,0 +1,82 @@
+import json
+import os
+from datetime import datetime
+
+import requests
+import xlsxwriter
+
+
+def fetch_data():
+    url = "https://ckan2.multimediagdansk.pl/gpsPositions?v=2"
+    response = requests.get(url)
+    if response.status_code == 200:
+        data = response.json()
+        parse_data(data_dir, data)
+    else:
+        print("Failed to retrieve data")
+
+
+def parse_data(data_dir: str, data: dict):
+    data_datetime = None
+    try:
+        data_datetime = datetime.strptime(data["lastUpdate"], "%Y-%m-%dT%H:%M:%SZ")
+    except:
```

Comment:
don't forghet specify which error shall be catched? KeyError in case of missing "lastUpdate" or ValueError if the date_string and format can’t be parsed
```suggestion
    except KeyError, ValueError:
```